src/node/stage/approved.rs

```diff
@@ -1895,318 +1806,62 @@ impl ApprovedPeer {
-        };
-
-        if bounce {
-            if let Some(sender) = sender {
-                debug!(
-                    "Unhandled message - bouncing: {:?}, hash: {:?}",
-                    msg,
-                    MessageHash::from_bytes(&msg_bytes)
-                );
-
-                self.send_bounce(&sender, msg_bytes);
-            } else {
-                trace!("Unhandled accumulated message, discarding: {:?}", msg);
-            }
-        } else {
-            debug!("Unhandled message from {:?}, discarding: {:?}", sender, msg);
-        }
-    }
-
-    fn is_message_handled(&self, msg: &MessageWithBytes) -> bool {
-        self.msg_filter.contains_incoming(msg)
-    }
-
-    fn set_message_handled(&mut self, msg: &MessageWithBytes) {
-        self.msg_filter.insert_incoming(msg)
-    }
-
-    fn relay_message(
-        &mut self,
-        _sender: Option<SocketAddr>,
-        message: &MessageWithBytes,
-    ) -> Result<()> {
-        self.send_signed_message(message)
-    }
-
-    fn should_handle_message(&self, msg: &Message) -> bool {
-        match &msg.variant {
-            Variant::BootstrapRequest(_)
-            | Variant::ParsecRequest(..)
-            | Variant::ParsecResponse(..)
-            | Variant::Bounce { .. } => true,
 
-            Variant::NeighbourInfo(_)
-            | Variant::UserMessage(_)
-            | Variant::AckMessage { .. }
-            | Variant::JoinRequest(_)
-            | Variant::MemberKnowledge(_) => self.chain.is_self_elder(),
-
-            Variant::GenesisUpdate(info) => {
-                !self.chain.is_self_elder() && self.is_genesis_update_new(info)
-            }
-            Variant::Relocate(_) => !self.chain.is_self_elder(),
-
-            Variant::MessageSignature(accumulating_msg) => {
-                match &accumulating_msg.content.variant {
-                    Variant::NeighbourInfo(_)
-                    | Variant::UserMessage(_)
-                    | Variant::NodeApproval(_)
-                    | Variant::AckMessage { .. }
-                    | Variant::Relocate(_) => true,
-
-                    Variant::GenesisUpdate(info) => {
-                        !self.chain.is_self_elder() && self.is_genesis_update_new(info)
-                    }
-
-                    // These variants are not be signature-accumulated
-                    Variant::MessageSignature(_)
-                    | Variant::BootstrapRequest(_)
-                    | Variant::BootstrapResponse(_)
-                    | Variant::JoinRequest(_)
-                    | Variant::MemberKnowledge(_)
-                    | Variant::ParsecRequest(..)
-                    | Variant::ParsecResponse(..)
-                    | Variant::Ping
-                    | Variant::Bounce { .. } => false,
-                }
-            }
-
-            Variant::BootstrapResponse(_) | Variant::NodeApproval(_) | Variant::Ping => false,
+    // TODO: move to Chain
+    fn print_rt_size(&self) {
```

Comment:
same as the other comment, try avoid mentioning the `routing table`, and any related, say `rt, table, ...`

Reply:
Same. In a followup PR.